README.md

```diff
@@ -71,15 +71,41 @@ cfg: { // Application Insights Configuration
 </script>
 ```
 
-The snippet supports reporting sdk script load failures as exceptions to the portal, this type of error would cause your application/website to either not report any telemetry or in some extreme cases become unstable due to excessive queuing of events which are never sent.
+> [!NOTE]
+> For readability and to reduce possible JavaScript errors, all of the possible configuration options are listed on a new line in snippet code above, if you don't want to change the value of a commented line it can be removed.
 
-It also supports some additional snippet specific configuration.
- - src (string) [required] - The URL of the SDK version to load
- - name (string) [optional] - The global SDK instance name to use, defaults to appInsights
- - ld (number in ms) [optional] - Defines the load delay to wait before attempting to load the SDK. Default value is 0ms and any negative value will just add a script tag to the page head and will block the page load event.
- - useXhr (boolean) [optional] - This setting is specifically for the reporting of SDK load failures. Reporting will first use fetch() if available and then fallback to XHR, setting this value to true just bypasses the fetch check. Use of this value would only be required if you know your app/site is being used in an environment where fetch would fail to send the failure events.
- - crossOrigin (string) [optional] - By including this setting the script tag used to download the SDK will include the crossOrigin attribute with this string value. When not defined (the default) no crossOrigin attribute is added. Recommended values are not defined (the default); ""; or "anonymous" (For all valid values see https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/crossorigin)
- - cfg (object) - This is the Application Insights configuration that is used to initialize your sdk.
+#### Reporting Script load failures
+
+This version of the snippet detects and reports failures when loading the SDK from the CDN as an exception to the Azure Monitor portal (under the failures &gt; exceptions &gt; browser), this exception provides visibility into failures of this type so that you are aware that your application is not reporting telemetry (or other exceptions) as expected. This signal is an important measurement in understanding that you have lost telemetry because the SDK did not load or initialize which can lead to:
```

Comment:
"`Failures` > `Exceptions`" for emphasis?

Reply:
re: "which can lead to:"
Clarity on whether or not SDK not loading actually causes the below issues. Or just limits observability on them

Reply:
I believe I've reworded as suggested